.pylintrc

```diff
@@ -82,3 +90,14 @@ disable=
 	# if:/else: and for:/else:.
 	useless-else-on-loop,
 	no-else-return,
+	# NOTE(lidiz): Python 3 make object inheritance default, but not PY2
+	useless-object-inheritance,
+	# NOTE(lidiz): Limit the length of function doesn't mean reducing
+	# complexity.
+	too-many-statements,
+	# NOTE(https://github.com/PyCQA/pylint/issues/1498)
+	unsubscriptable-object,
```

Comment:
I only see this failing on tests, and only in one file: _reflection_servicer_test.py. There is no need to disable this globally for non-test code, and even for tests, let's disable it on a per-file or per-line basis.